Classes/Manager/FLEXManager+Private.h

```diff
@@ -13,4 +13,7 @@
 /// An array of FLEXGlobalsTableViewControllerEntry objects that have been registered by the user.
 @property (nonatomic, readonly, strong) NSArray *userGlobalEntries;
 
+/// An array of FLEXCustomContentTypeViewer objects that have been registered by the user.
+@property (nonatomic, readonly, strong) NSMutableArray *customContentTypeViewers;
+
```

Comment:
`customContentTypeViewers` should be exposed as `NSArray` even in `FLEXManager+Private.h`